editor.planx.uk/src/@planx/components/FindProperty/Public/Map.tsx

```diff
@@ -126,7 +126,7 @@ export default function PlotNewAddress(props: PlotNewAddressProps): FCReturn {
           drawType="Point"
           geojsonData={JSON.stringify(props?.boundary)}
           geojsonColor="#efefef"
-          geojsonBuffer="10"
+          geojsonBuffer={10}
```

Comment:
Should have spotted this on the map PR sorry - but would this be a breaking change for the map component?

I'm very certain it's only us using this prop currently so it's not a blocker here right now, but maybe we should have gone to v0.8.0

I'd be happy with continuing as-is and we can maybe just let BOPS know tomorrow to be aware if they're relying on this. Linting should catch it anyway before it causes too many headaches!

Reply:
Thanks for the explanation - agreed that it's helpful yet headachey 😅